src/pages/SetupPlot.vue

```diff
@@ -309,6 +301,15 @@ export default defineComponent({
       if (result) {
         this.store.setPlotPath(result);
       }
+
```

Comment:
I see 2 problems with this approach. 
1- there are still `red` lines in the console, see my above comment (case 2: when folder icon is clicked although there is no `plot` file created)
2- this function now has the privilege to create folders, and this creation operation is applied after the user selects directory... If we are not in the `subspace-desktop` folder, I think we should not create any folders without prompting the user. 

My suggestion would be: moving this chunk of code into the `mounted()` part, or maybe creating a separate function for these chunk of code and calling that function in the `mounted()` part, so that we know we will only create the `plot` folder, nothing else. And other functions will stay clean